globus_automate_client/flows_client.py

```diff
@@ -285,31 +304,32 @@ def deploy_flow(
         :param validate_schema: Set to ``True`` to validate the provided
             ``input_schema`` before attempting to deploy the Flow.
 
+        :param dry_run: Set to ``True`` to test whether the Flow can be
+            deployed successfully.
+
         """
         if validate_definition:
             validate_flow_definition(flow_definition)
         if validate_schema:
             validate_input_schema(input_schema)
-        self.authorizer = self.flow_management_authorizer
-        temp_body: Dict[str, Any] = {"definition": flow_definition, "title": title}
-        temp_body["subtitle"] = subtitle
-        temp_body["description"] = description
-        temp_body["keywords"] = keywords
-        # We'll accept some aliases for the role lists passed in kwargs
-        temp_body["flow_viewers"] = merge_lists(
-            flow_viewers, kwargs, "visible_to", "viewers"
-        )
-        temp_body["flow_starters"] = merge_lists(
-            flow_starters, kwargs, "runnable_by", "starters"
-        )
-        temp_body["flow_administrators"] = merge_lists(
-            flow_administrators, kwargs, "administered_by", "administrators"
-        )
-        temp_body["subscription_id"] = subscription_id
-        # Remove None / empty list items from the temp_body
+        temp_body: Dict[str, Any] = {
```

Comment:
Wow, this is massively more readable to me. Thanks for all this miscellaneous cleanup!

Reply:
My pleasure!